markhor_tracks/src/markhor_hw_interface.cpp

```diff
@@ -79,9 +105,10 @@ void MarkhorHWInterface::setupCTREDrive()
     front_left_drive->Config_kF(0, 0, timeout_ms_);
```

Comment:
By curiosity, what's a kF and would it be possible that we could use it one day? It seems out of place being the only one hard coded.

Reply:
KF is Velocity Feed Forward, from what I understands, it helps to compensate for the fact you need to push harder the faster you go, it seems to be a gain multiplying the speed of the axis, see https://deltamotion.com/support/webhelp/rmctools/Registers/Register_Descriptions/Axis_Parameter_Registers/Position_Velocity_Control/Param_Velocity_Feed_Forward_Gain.htm

I don't think we need this as the performance is currently sufficient